python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py

```diff
@@ -200,6 +200,7 @@ def filtered_vrfs(
             vrf.l3_interfaces = vrf.l3_interfaces._filtered(
                 lambda l3_interface: bool(self.hostname in l3_interface.nodes and l3_interface.ip_addresses and l3_interface.interfaces)
             )
+            vrf.l3_port_channels = vrf.l3_port_channels._filtered(lambda l3_port_channel: bool(self.hostname == l3_port_channel.node and l3_port_channel.name))
```

Comment:
@ClausHolbechArista I think we can remove the check for name as it is a required key:

```suggestion
            vrf.l3_port_channels = vrf.l3_port_channels._filtered(lambda l3_port_channel: bool(self.hostname == l3_port_channel.node))
```